docs/custom_resources.adoc

```diff
@@ -1,17 +1,22 @@
 = RHOAS Operator Custom Resources
 
-The operator manages the following resources:
- - `CloudServicesRequest`
- - `KafkaConnection`
- - `CloudServiceAccountRequest`.
- 
-All resources are namespace scoped and include operator-managed conditions. 
+The OpenShift Application Sercvices (RHOAS) Operator manages the following types of Custom Resources (CRs):
 
+* `CloudServicesRequest`
+* `KafkaConnection`
+* `CloudServiceAccountRequest`
+
+All CRs are namespace-scoped and include Operator-managed conditions. Each CR is described in the sections that follow.
+
+[id="CloudServicesRequest"]
 == CloudServicesRequest
 
-This custom resource is populated with services available to the provided access token. These services are listed under the status object. This resource is used to get a list of available services. For instance, the id of a Kafka service is used by `KafkaConnection`.
+The `CloudServicesRequest` CR is used to get a list of available services. The controller for the CR updates the CR with services that are available to the provided access token. These services are listed under the `status` subresource. For example, the ID of an available Kafka service is used by the `KafkaConnection` CR.
 
 === Example
+
+An example of the `CloudServicesRequest` CR is shown below.
```

Comment:
```suggestion
An example of the `CloudServicesRequest` Custom Resource
```

Reply:
I think this change treats the text as a caption for the code block, which it isn't. If it's OK, I think it's better to leave as-is.

Reply:
Agreed.